src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -525,7 +534,7 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                 // If falseValue is zero, just zero out those lanes of targetReg using `movprfx`
                                 // and /Z
                                 GetEmitter()->emitIns_R_R_R(INS_sve_movprfx, emitSize, targetReg, maskReg, targetReg,
```

Comment:
We shouldn't be using `soptEmb` for `movprfx`

Reply:
Good catch, we should definitely not be.